superdesk/publish/formatters/ninjs_ftp_formatter.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+from .ninjs_formatter import NINJS2Formatter
+from superdesk.storage.amazon_media_storage import _guess_extension
```

Comment:
imo would be good to move this into some `media_utils` module

Reply:
I've moved the functions off into the media module